environments/production/public.yml

```diff
@@ -172,14 +172,14 @@ localsettings:
   # When to consider multiplex settings
   ES_MULTIPLEX_TO_VERSION: '6'
   # Index Multiplexer Settings  
-  ES_APPS_INDEX_MULTIPLEXED: False
+  ES_APPS_INDEX_MULTIPLEXED: True
   ES_CASE_SEARCH_INDEX_MULTIPLEXED: False # Index swapped
```

Comment:
Why not this one?

Reply:
This has already been reindexed and swapped during the BHA work. I have added it to the [commit](https://github.com/dimagi/commcare-cloud/pull/6443/commits/7d14ba3bc1b9862c03522babcfb6af479d2487ae) message as well.